frontend/src/app/components/Loading.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 
 const Loading = () => {
-  return (
+  return <div className="full-page-wrapper centered overflow-hidden">
```

Comment:
This just lets us use the same <Loading /> component in a few places, instead of having the same structure copied into a few files as it was before.